include/stp/Util/Attributes.h

```diff
@@ -57,4 +57,11 @@ THE SOFTWARE.
     #define DLL_LOCAL
 #endif
 
+// Provide thread safety if possible
+#if __cplusplus >= 201103L
+    #define THREAD_LOCAL thread_local
+#else
+    #define THREAD_LOCAL
```

Comment:
We use C++11 so thread_local is there. If it's not there, setting it empty is probably a bad idea, as for some it would fail to run multi-threaded, and for some it would not... leading to very hairy debugging issues :O

Reply:
Ok, thanks. Then in that case maybe we should just use `thread_local` directly instead of using the macro.  There are a couple of places where I inserted #include "stp/Util/Attributes.h" in order to get the macro definition (lib/extlib-abc/aig.h, lib/extlib-abc/aig/kit/kitTruth.c, and lib/extlib-constbv/constantbv.cpp) those can be removed too.